### PR TITLE
feat: add embedding metadata type

### PR DESCRIPTION
**Pull request by cabreraalex**
# Description
Add embedding as a column type for later work on similarity search, etc.

### CODE REVIEW

**Comment by cabreraalex** [comment]
We likely want to also have a PROJECTION type that is 2D numeric vectors - it's harder to detect this though, no explicit type in the arrow schema. Would have to wait to have the data later and query the DB, do some inference.

**Comment by cabreraalex** [comment]
Could also do this dynamically where when you go to projection tab we get all the embedding cols and see which are 2D

**Comment by Sparkier** [comment]
> Could also do this dynamically where when you go to projection tab we get all the embedding cols and see which are 2D

I like this idea.

File: frontend/src/lib/util/util.ts [hunk 1]
```diff
@@ -42,18 +42,16 @@ export function getProjectRouteFromURL(url: URL) {
 
 function columnTypeOrder(colType: ZenoColumnType) {
 	switch (colType) {
-		case ZenoColumnType.FEATURE:
+		case ZenoColumnType.ID:
```
Comment (Sparkier) [inline frontend/src/lib/util/util.ts:45]: Don't think this makes sense in many cases. Used this for putting feature columns up top in the selects.

**Review by Sparkier — APPROVED** [review]

**Pull request merged**